[EDU-4521] refactor: add tabs to Marketplace Integrations guides (#1172)
aziontech/docs: src/content/docs/en/pages/guides/marketplace/integrations/how-to-massive-redirection.mdx

To create an edge application, go to the [building](/en/documentation/products/guides/build/build-an-application/) section.
:::

### Configuring Edge Firewall

To configure your integration, proceed as follows:

<Tabs client:visible>
    <Fragment slot="tab.console">Console</Fragment>
    <Fragment slot="tab.rtm">Real-Time Manager</Fragment>

<Fragment slot="panel.console">
1. On the upper-left corner, select **Products menu** > **Edge Application** on the **BUILD** section.
2. On the listing page of your edge applications, select the one you want to use with the integration.
3. On the application page, on the **Main Settings** tab, locate the **Edge Functions** switch and turn it on.
4. Click the **Save** button.

> You'll receive a successful message indicating you have updated your edge application.

Author: ```suggestion
You'll receive a success message indicating that your edge application has been updated.
```

</Fragment>

<Fragment slot="panel.rtm">
1. On the upper-left corner, select **Products menu** > **Edge Application** on the **BUILD** section.
2. On the listing page of your edge applications, select the one you want to use with the integration.
3. On the application page, on the **Main Settings** tab, locate the **Edge Functions** switch and turn it on.
4. Click the **Save** button.

> You'll receive a successful message indicating you have updated your edge application.

Author: ```suggestion
You'll receive a success message indicating that your edge application has been updated.
```
